From 980b86a44ac0129cb5871fb94b66233df78a35cf Mon Sep 17 00:00:00 2001
From: Matej Habrnal <mhabrnal@redhat.com>
Date: Mon, 9 Nov 2015 15:02:36 +0100
Subject: [PATCH] uploader: allow empty username and password

Due to added the possibility to configure SSH keys the username and the
password may not be set. Also there is an another reason why we decided to
allow those options empty. The username and the password can be defined within
URL and if they are not provided either by conf file or within URL,
reporter-uploader will ask for that.

Related to rhbz#1289513

Signed-off-by: Matej Habrnal <mhabrnal@redhat.com>
---
 src/plugins/report_Uploader.xml.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/plugins/report_Uploader.xml.in b/src/plugins/report_Uploader.xml.in
index 7df006a..8d648eb 100644
--- a/src/plugins/report_Uploader.xml.in
+++ b/src/plugins/report_Uploader.xml.in
@@ -21,12 +21,12 @@
         </option>
         <option type="text" name="Upload_Username">
             <_label>User name</_label>
-            <allow-empty>no</allow-empty>
+            <allow-empty>yes</allow-empty>
             <_description>Use this field if you do not want to have user name in URL</_description>
         </option>
         <option type="password" name="Upload_Password">
             <_label>Password</_label>
-            <allow-empty>no</allow-empty>
+            <allow-empty>yes</allow-empty>
             <_description>Use this field if you do not want to have password in URL</_description>
         </option>
         <advanced-options>
-- 
1.8.3.1

